site/src/api/queries/workspaces.ts

```diff
@@ -0,0 +1,82 @@
+import * as API from "api/api";
```

Comment:
Quick heads up: I added a workspaces file for a PR I merged yesterday:
https://github.com/coder/coder/pull/10142/files#diff-6de11df3ac825274048e85bb42fc44a1cd5ee8630dc0d547f4cb2ee0191f31a5